apps/charterafrica/app.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "charterafrica",
+  "scripts": {
+    "predeploy": "export NEXT_PUBLIC_APP_URL=$(dokku config:get charterafrica NEXT_PUBLIC_APP_URL) && export RESOURCES_SECRET_TOKEN=$(dokku config:get charterafrica RESOURCES_SECRET_TOKEN)",
+    "update-ecosystem-list": "curl -X POST -H \"x-api-key: $RESOURCES_SECRET_TOKEN\" $NEXT_PUBLIC_APP_URL/api/v1/resources/ecosystem/entities",
+    "update-ecosystem-content": "curl -X PUT -H \"x-api-key: $RESOURCES_SECRET_TOKEN\" $NEXT_PUBLIC_APP_URL/api/v1/resources/ecosystem/entities"
+  },
+  "cron": [
+    {
+      "command": "npm run update-ecosystem-content",
+      "schedule": "@daily"
```

Comment:
Not sure there is but let me ask anyway: is there anyway of externalising schedule e.g. an env var? I'm trying to avoid the situation where we have change the code to just make a new schedule.

Reply:
I doubt if that's possible unless @thepsalmist has tested if this is possible. Documentation doesn't  seem to specify this